dstep/translator/Context.d

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Mar 21, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+
+module dstep.translator.Context;
+
+import mambo.core._ : isBlank;
```

Comment:
The `_` module imports all modules in the core package. Feels a bit weird to do selective imports on that.